### ISSUE TITLE
Any plans to make a one-time payment option for TagSpaces Pro?

### ISSUE THREAD

**Issue by proxycyber** [tagspaces/tagspaces#625]
I mean this in the nicest possible way because I love TagSpaces, but I don't see a reason to pay to use it every year. It's a text editor, there are open-source software that do similar things to this that are more frequently updated, free, offer end-to-end encryption, file syncing, taggeration, etc.

Now, I'm not saying I wouldn't pay, but I just don't see a reason to pay for software that has so many flaws (by that I mean bugs) to begin with, just in the free version.

Again, this isn't meant to sound rude and I hope you get that. But it's just I don't get why this would be a paid app, as it does nothing I haven't already seen before. In fact, it probably has less features than other free and more powerful alternatives. Granted I prefer the layout of this app and I'd be willing to pay, but I don't think I'd pay $45 for something like a text editor. You can correct me if I'm wrong about the price. Nothing about this app is worth that much, but I get it because it's probably one person doing it who has very little time on his hands and who is learning to code. Still, the business tactic is illogical. The cheaper the product, the more likely it'll be for a higher volume of purchases. Cloud storage and other office suites are cheaper... at like $3 a month to $10 or $5 once forever.

I think it'd be nice if there was a pay once and keep it forever option, rather than a subscription method. It makes me lose so much faith when developers do this because it wasn't a paid thing in the beginning and suddenly is. But I guess down the line, hopefully this will be a one-time payment of like $4 or $10 or something. Again, I absolutely love TagSpaces, I just don't see a reason to pay to use it.

**Comment by uggrock** [comment]
For now we do not have such plans, please consider paying for the PRO version as a way to support the development of the free version. The price comparison is not realistic, the majority of the features are available for free.

**Comment by proxycyber** [comment]
I agree, TagSpace's price is not realistic. as there are far more advanced editors that are far cheaper. I totally support projects that are for Linux or that are open-source, but I just don't think it's worth that much at the moment. I will definitely consider paying once the price is more realistic. I do think you do decent, and I absolutely love the concept. Hopefully in the future, I will pay if there's a one-time payment of about $9.99 or $12.

**Comment by nmaggioni** [comment]
@uggrock Take [Plex's prices](https://www.plex.tv/features/plex-pass/) for example: they ask for your same yearly amount, but they are offering a service that you constantly take advantage of and that **uses external resources to work**.

[Sublime Text](https://www.sublimetext.com/buy), on the other hand, charges a one-time fee as most software products: you buy the license and use the software forever, as it is not based on external resources like Plex is.

In my humble opinion your software is a mix between Sublime Text's and [WinRAR](https://www.win-rar.com/buy/index_shop.php?AffID=200001006)'s pricing models, and as such is way more suitable to a one-time payment rather than a subscription. I'd be happy to make a single €39 payment - maybe even a little more - as TagSpaces is becoming an important tool for me ( :smiley: ), but at the same time a repeated payment is simply not justifiable for my usage model.

I suppose that I am an average user when it comes to free vs. paid software, and as such I think _(and that's my own, personal opinion)_ that you could attract way more paid users with a one-time license. It could as well be a per-host license if you prefer, instead of a per-user one - my assumptions above still make sense to me.

**TL;DR:** One time payment of €39 or a little more? Sure, just tell me where to click! Yearly license at the same price? Thank you, but I will stick to the free version.
